Quality.md

```diff
@@ -0,0 +1,31 @@
+## Quality Assistance
+
+# Quality Coach
+
+> I work with Octopus team members to coach them to deliver better overall quality.
+
+### I am a coach
+
+- I coach teams to deliver a product that is user-friendly, efficient, consistent, robust, and useful to customers.
+- I can manage difficult conversations and tactfully challenge others, leaving them with a positive impression of myself and Octopus.
+- I can effectively coach people and pass on my knowledge.
+- I help teams to achieve better business value.
+- I recognize and accept that change is constant, and my approach allows me to tactfully challenge, or confidently adapt, depending on which I deem appropriate.
+- I possess writing skills that let me be clear, concise, articulate, and persuasive in all of my outgoing communications.
+- I know how to make pragmatic decisions in order to maintain influence.
+
+### I ensure Octopus teams can ship with confidence
+
+- I am an advocate for quality.
+- I help engineers refine requirements and identify the right level of testing
+- I use customer’s feedback and data to measure our level of quality.
+- I identify gaps in the quality processes and collaborate with the team to close those gaps.
+- I introduce new testing practices for faster feedback cycles.
+
```

Comment:
I help identify the right team for the unresolved ambiguous tasks. Sometimes bugs(P3s and P4s) get lost while in development cycle as they might existing issues. Trying to find a board to fix those things and see if there are any customer issues around it.

Reply:
I help engineers map requirements to user scenarios and to identify acceptance criteria. Suggest test automation coverage based on this analysis.

Reply:
I work closely with products, designers, engineers, tech writers, and schools at octopus to ensure usability of the product.

Reply:
> I help identify the right team for the unresolved ambiguous tasks.
That team is the Product team (MichaelR/MatthewC) and the QRF backlog

Reply:
> I help engineers map requirements to user scenarios and to identify acceptance criteria. Suggest test automation coverage based on this analysis.

Would this not be covered under `refine requirements and identify the optimal kind, level, and timing for testing.`?